templates/web/pages/session/signed_in.tt.html

```diff
@@ -34,6 +34,15 @@ <h3>[% lang("resources") %]</h3>
                 [% lang('faq_for_producers') %]
             </a>
         </li>
+        <li>
```

Comment:
It looks like we have repeated code in signed_in.tt.html and site_layout.tt.html

Could you create a separate file called producers_resources_list that contains the 3 <li>..</li> and then include that file in the other 2?

Reply:
Okay..try doing it..